config.yaml.defaults

```diff
@@ -718,3 +718,12 @@ classifications:
     "variable": "#3a87ad",
     "Time-domain Source": "#ff0000",
   }
+
+gcnTags:
```

Comment:
Are these color codes? If so perhaps better to name this config key something more specific?

Reply:
I agree--this whole section could be called "colors". Ie.

```yaml
colors:
   classifications:
        variable: #xxxx
         ....
    gcn_tags:
         BNS: #xxxx
```